app/src/main/java/org/simple/clinic/home/patients/PatientsTabModel.kt

```diff
@@ -37,9 +37,6 @@ data class PatientsTabModel(
   val hasLoadedUser: Boolean
     get() = user != null
 
-  val hasLoadedNumberOfPatientsRegistered: Boolean
-    get() = numberOfPatientsRegistered != null
```

Comment:
`numberOfPatientsRegistered` parameter is unused. Let's get rid of that as well.